### PR TITLE
Updating release notes for 2023.06.25-2023.07.02

### PR DESCRIPTION
**Pull request by sabbour**

### CODE REVIEW

**Comment by wangyira** [comment]
Want to add [this doc](https://learn.microsoft.com/en-us/azure/aks/open-ai-quickstart) to the release notes, checking it with the v-team, will send a blurb shortly. Thanks

**Comment by wangyira** [comment]
> Want to add [this doc](https://learn.microsoft.com/en-us/azure/aks/open-ai-quickstart) to the release notes, checking it with the v-team, will send a blurb shortly. Thanks

Could you please add this to the announcement, thank you!

Learn [how to deploy an application that uses OpenAI on AKS](https://learn.microsoft.com/azure/aks/open-ai-quickstart) by running a sample multi-container solution for an e-commerce website and deploying OpenAI service to generate product descriptions.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-07-02
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Windows 2019 will be retired in Kubernetes v1.33 and above (ETA March 2026). Customers should [upgrade to Windows 2022](https://learn.microsoft.com/azure/aks/upgrade-windows-2019-2022).
+* Kubernetes 1.24 is being deprecated end of July 2023. From Kubernetes 1.25 the default cgroups implementation on AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance.
```
Comment (allyford) [inline CHANGELOG.md:10]: 1.24 will be retired from regular support but will be supported via our platform support policy. See documentation here: https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy:~:text=11*%2C%201.16.*10*-,Platform%20support%20policy,-Platform%20support%20policy

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+* Preview Features
+  * Added Prometheus scraping annotations to App Routing add-on.
+  * Support for Azure DNS private zones in the App Routing add-on.
+  * Support for changing pod CIDR for bring your own CNI plugin.
+
+* Behavior Changes
+  * The default OS disk type for non-ephemeral OS disks is now Standard SSD.
+  * Enable on-call engineers to access Open Service Mesh (OSM) CRDs for debugging.
+
+* Bug Fixes
```
Comment (allyford) [inline CHANGELOG.md:24]: Anything customers need to do to get these fixes?

Comment (sabbour) [reply CHANGELOG.md:25]: Not for these 2 fixes.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-07-02
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Windows 2019 will be retired in Kubernetes v1.33 and above (ETA March 2026). Customers should [upgrade to Windows 2022](https://learn.microsoft.com/azure/aks/upgrade-windows-2019-2022).
+* Kubernetes 1.24 is being deprecated end of July 2023 and support will transition to our [platform support policy](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy).
```
Comment (allyford) [inline CHANGELOG.md:10]:
```suggestion
* Kubernetes 1.24 is being deprecated end of July 2023 and support will transition to our [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy). 
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
+* Kubernetes 1.24 is being deprecated end of July 2023 and support will transition to our [platform support policy](https://learn.microsoft.com/en-us/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy). 
+* Starting Kubernetes 1.25, the default cgroups implementation on AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance.
+* A [known issue](https://github.com/Azure/AKS/issues/3718) in Kubernetes version 1.24 is causing name resolution failures in Windows pods. Customers experiencing this issue should upgrade their cluster to Kubernetes version 1.25.
+* Learn [how to deploy an application that uses OpenAI on AKS](https://learn.microsoft.com/en-us/azure/aks/open-ai-quickstart?tabs=aoai) by running a sample multi-container solution for an e-commerce website and deploying OpenAI service to generate product descriptions.
```
Comment (shashankbarsin) [inline CHANGELOG.md:13]:
```suggestion
* Learn [how to deploy an application that uses OpenAI on AKS](https://learn.microsoft.com/azure/aks/open-ai-quickstart?tabs=aoai) by running a sample multi-container solution for an e-commerce website and deploying OpenAI service to generate product descriptions.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,41 @@
...
+
+### Release notes
+
+* Preview Features
+  * Added Prometheus scraping annotations to App Routing add-on.
```
Comment (shashankbarsin) [inline CHANGELOG.md:18]: Prometheus scraping annotations are means to the end. We need to reword this to convey what the user gets as benefit... For example, metrics under prometheus on nginx used in WAR addon. Does it work only with managed prometheus?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,41 @@
...
+  * Support for changing pod CIDR for bring your own CNI plugin.
```
Comment (shashankbarsin) [inline CHANGELOG.md:19]: docs link?

File: CHANGELOG.md (hunk 5 above)
Comment (sabbour) [reply CHANGELOG.md:18]: Works with any Prometheus. Will add the benefit.

File: CHANGELOG.md (hunk 6 above)
Comment (sabbour) [reply CHANGELOG.md]: Updated.

File: CHANGELOG.md (hunk 4 above)
Comment (sabbour) [reply CHANGELOG.md]: Removed the link.

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,39 @@
...
+* Kubernetes 1.24 is being deprecated end of July 2023 and support will transition to our [platform support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#platform-support-policy). 
+* Starting Kubernetes 1.25, the default cgroups implementation on AKS nodes will be cgroupsv2. Older versions of Java, .NET and NodeJS do not support memory querying v2 memory constraints and this will lead to out of memory (OOM) issues for workloads. Please test your applications for cgroupsv2 compliance.
+* A [known issue](https://github.com/Azure/AKS/issues/3718) in Kubernetes version 1.24 is causing name resolution failures in Windows pods. Customers experiencing this issue should upgrade their cluster to Kubernetes version 1.25.
+
+### Release notes
+
+* Preview Features
+  * Added annotations to App Routing add-on for Prometheus automatic discovery and scraping of the [nginx ingress controller metrics](https://kubernetes.github.io/ingress-nginx/user-guide/monitoring/).
+  * Support for [changing pod CIDR](https://learn.microsoft.com/en-us/cli/azure/aks?view=azure-cli-latest#az-aks-update) for bring your own CNI plugin.
```
Comment (shashankbarsin) [inline CHANGELOG.md:18]:
```suggestion
  * Support for [changing pod CIDR](https://learn.microsoft.com/cli/azure/aks?view=azure-cli-latest#az-aks-update) for bring your own CNI plugin.
```

**Review by allyford — APPROVED** [review]

**Review by shashankbarsin — APPROVED** [review]
